src/panel/poi/PoiPanel.jsx

```diff
@@ -56,6 +57,13 @@ export default class PoiPanel extends React.Component {
         this.setState({ isPoiInFavorite });
       }
     });
+
+    // Show return arrow on mobile if user comes from PoI / favorites list
```

Comment:
As always, we should avoid manipulating DOM directly in React if possible, especially stuff outside the component render tree.
Some time ago I centralized every top bar manipulations in `PanelManager`, especially in [this method](https://github.com/Qwant/erdapfel/blob/master/src/panel/PanelManager.jsx#L89-L105), to limit them and prepare the way for the top bar migration to React.

As a result, we should find how to do all this (this block and the two others below) in `PanelManager`.